components/cli/cmd/cellery/init.go

```diff
@@ -25,12 +25,18 @@ import (
 )
 
 func newInitCommand() *cobra.Command {
+	var projectName = ""
 	cmd := &cobra.Command{
 		Use:   "init",
```

Comment:
Please update the usages to have the optional project argument

Reply:
Updated.